std/uni.d

```diff
@@ -8324,15 +8332,15 @@ auto asUpperCase(Range)(Range str)
     assert("hEllo".asUpperCase.equal("HELLO"));
 }
 
-// explicitly undocumented
+/// ditto
 auto asLowerCase(Range)(auto ref Range str)
```

Comment:
@MartinNowak git says that you made these overloads. Why didn't you document them? I don't see why this should be hidden from the user.